sync_diff_inspector/source/basic.go

```diff
@@ -99,9 +98,19 @@ func getOriginTable(sourceTableMap map[string]*common.TableSource, table *common
 	return originSchema, originTable
 }
 
+func (s *BasicSource) GetSourceTableMap() map[string]*common.TableSource {
```

Comment:
why add this function?

Reply:
We can implement the `GetTableStruct` in each Source. `SourceTableMap` is too implied

Reply:
Fixed